packages/replayio/src/utils/installation/getLatestReleases.ts

```diff
@@ -5,24 +5,24 @@ import { runtimeMetadata } from "./config";
 import { debug } from "./debug";
 import { Release } from "./types";
 
-const { platform, runtime } = runtimeMetadata;
+const { platform, runtime, architecture } = runtimeMetadata;
 
 export async function getLatestRelease() {
   debug("Fetching release metadata");
 
   const response = await fetch(`${replayAppHost}/api/releases`);
   const json = (await response.json()) as Release[];
-  const latestRelease = json.find(release => {
-    if (release.platform === platform && release.runtime === runtime) {
-      if (platform === "macOS") {
-        return process.arch.startsWith("arm") === release.releaseFile.includes("arm");
-      } else {
-        return true;
-      }
-    }
-  });
+  const latestRelease = json.find(
+    release =>
+      release.platform === platform &&
+      release.runtime === runtime &&
+      (release.architecture === architecture || release.architecture === null)
+  );
```

Comment:
could we say that it will be possible to remove `|| release.architecture === null` from here once the new releases with the architecture field get published?

Reply:
Yes. The CLI only checks for the _latest_ release. So once the new releases go out then we could remove this.